pcs/resource.py

```diff
@@ -771,6 +808,7 @@ def resource_move(lib: Any, argv: List[str], modifiers: InputModifiers):
     if not argv:
         raise CmdLineInputError("must specify a resource to move")
     resource_id = argv.pop(0)
+    _check_is_not_stonith(lib, [resource_id])
```

Comment:
Can this be moved to a library command? Then we could simply replace deprecation warning with an error in order to actually forbid moving stonith resources.